CHANGELOG.md

```diff
@@ -210,14 +210,14 @@ UPGRADE:
 * For NGINX Plus, please build your own image using the 1.9.0 source code.
 * For Helm, use version 0.7.0 of the chart.
 
```

Comment:
no need to remove the -use-ingress-class-only mentions from the changelog

Reply:
done